### ISSUE TITLE
Exclamation mark with no k.o. hit (damage display bug?)

### ISSUE THREAD

**Issue by Alua7** [MoArtis/RCGMod#30]
I hit a tiger("King") with an yoyo (Misako) and it displayed 36! But no k.o. Bombic bra is on but hit did not a knockout.
UPD: hmm unlike other weapons yoyo seems not to actually crit. So maybe game's bug
Also: yoyo gives ! but not crit, other weapons give crit but no "!".
btw funny bombic bra also affects cars

**Comment by MoArtis** [comment]
Seems to be working fine with the weapon I tried so far.
![image](https://user-images.githubusercontent.com/3904610/73186890-76d94000-415b-11ea-9f4b-21e7e738ea5b.png)
And yeah Bomb bra affects anything that can damage with a direct hit. (notice that Bomb bra doesn't work when throwing enemies and weapons.)
I'm on my way to test the yoyo.

**Comment by MoArtis** [comment]
![image](https://user-images.githubusercontent.com/3904610/73190197-9161e800-4160-11ea-9971-78af9e531297.png)
Same for the yoyo. I will leave it like that for now. We will see if the bug is still reproducible with the 1.3. But to be clear I changed nothing in the code about that. At least for now. 😁

**Comment by Alua7** [comment]
https://youtu.be/jA43RozSMlA?t=15s
Pls start from 0:15.
1 episode - ! opp dies (but I swear it's not crit)
2 episode - ! opp doesn't die
3 episode - crit with no !
4 episode - one more ! 
I got ! on various situations - 1st hit, 5th hit but no k.o.
Maybe troubles of old version tho)

**Comment by MoArtis** [comment]
I think I found the root cause of the issue. Not sure why I did it that way back then but It seemed consistent without modifying RCG's code. But reading the code again, it seems wrong. I changed the code to ensure more consistent results.
A test with a 60% chance of triggering the effect seems conclusive:
![image](https://user-images.githubusercontent.com/3904610/73247813-ae43fd00-41ec-11ea-99e1-2b423b18d9b2.png)
The fix will be part of the version 1.3.
